src/styles/blocks/product.scss

```diff
@@ -0,0 +1,104 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
```

Comment:
This value was already used for margin or padding, isn't it? let's create a variable